src/app/dashboard/dashboard.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, inject } from '@angular/core';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `inject` function is not imported from the correct module. It should be imported from `@angular/core`. Also, it's not recommended to use `inject` outside of the testing environment. Instead, you should use dependency injection in the constructor.
```diff
- import { Component, inject } from '@angular/core';
+ import { Component } from '@angular/core';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->